src/app/attachment/save-action/inbox-linshare-attachment-save-action.controller.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+angular.module('linagora.esn.unifiedinbox.linshare')
+  .controller('inboxLinshareAttachmentSaveActionController', inboxLinshareAttachmentSaveActionController);
+
+function inboxLinshareAttachmentSaveActionController(
+  $scope,
+  asyncAction,
+  esnConfig,
+  linshareApiClient,
+  inboxLinshareAttachmentSaveActionService,
```

Comment:
Do we need explicit dependencies here?

Reply:
There'll be another PR to handle explicit dependencies & translation issues.